app/src/store/data.ts

```diff
@@ -159,6 +161,28 @@ export default function useDataStore() {
     lastBlockTimestamp.value = (timestamp as BigNumber).toNumber();
     grants.value = grantsList as Grant[];
     grantRounds.value = grantRoundsList as GrantRound[];
+    const metaPtrs = grants.value.map((grant) => grant.metaPtr);
+    const newMetadata = metaPtrs
+      .filter((metaPtr) => !grantMetadata.value[metaPtr]) // only want new metaPtrs
```

Comment:
Just double checking -> it looks like a grant can have multiple metadata but we want only the which as status as pending ?
not super clear on what we are doing here

Reply:
i'll try to add some comments!